langchain4j-vearch/src/main/java/dev/langchain4j/store/embedding/vearch/VearchEmbeddingStore.java

```diff
@@ -0,0 +1,257 @@
+package dev.langchain4j.store.embedding.vearch;
+
+import dev.langchain4j.data.document.Metadata;
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.store.embedding.CosineSimilarity;
+import dev.langchain4j.store.embedding.EmbeddingMatch;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.RelevanceScore;
+
+import java.time.Duration;
+import java.util.*;
+
+import static dev.langchain4j.internal.Utils.*;
+import static dev.langchain4j.internal.ValidationUtils.*;
+import static java.time.Duration.ofSeconds;
+import static java.util.Collections.singletonList;
+import static java.util.stream.Collectors.toList;
+
+public class VearchEmbeddingStore implements EmbeddingStore<TextSegment> {
+
+    private final VearchConfig vearchConfig;
+    private final VearchClient vearchClient;
+    private final boolean normalizeEmbedding;
+
+    public VearchEmbeddingStore(String baseUrl,
+                                Duration timeout,
+                                VearchConfig vearchConfig,
+                                Boolean normalizeEmbedding) {
```

Comment:
```suggestion
                                Boolean normalizeEmbeddings) {
```
WDYT? Please also document this behaviour.

Reply:
Nice! document also added